config/routes.rb

```diff
@@ -74,17 +75,13 @@
         end
       end
 
-      resources :criteria do
+      resources :criteria, except: [:index] do
         collection do
           post 'update_positions'
         end
       end
 
-      resources :rubric_criteria do
-        member do
-          delete 'destroy'
-        end
-
+      resources :rubric_criteria, only: [:index] do
```

Comment:
The controller grabs the criteria for a particular assignment. We use this info in the marking scheme page to display  all criteria with their respective maximum mark and visibility. We were not able to merge both index pages because they have different upload/download links. If you want I can try migrating the index action into the criteria controller and create partials for the links.